scripts/jupyter_gadi.sh

```diff
@@ -82,7 +90,7 @@ if [ -z "$MEM" ]; then
     MEM="$(( NCPUS * 4 ))gb"
 fi
 
-SUBMITOPTS="-N jupyter-notebook $PROJECT -q '$QUEUE' -l 'ncpus=${NCPUS},mem=${MEM},walltime=${WALLTIME},jobfs=${JOBFS}'"
+SUBMITOPTS="-N jupyter-notebook -P $PROJECT -q '$QUEUE' -l 'storage=${STORAGE},ncpus=${NCPUS},mem=${MEM},walltime=${WALLTIME},jobfs=${JOBFS}'"
```

Comment:
See https://github.com/coecms/nci_scripts/blob/master/gadi_jupyter#L139 for an alternate way to set up storage, which just asks for everything that's available.

This looks fine for manually assigning.